CHANGELOG.md

```diff
@@ -2,7 +2,11 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-_(none)_
+* Upgrade to pulumi-terraform-bridge v2.7.3
+* Upgrade to Pulumi v2.9.0, which adds type annotations and input/output classes to Python  
+  **Please Note:**  
+  The NodeJS and Go SDKs have lowercased all provider modules. Therefore, `azuredevops.Core` will
+  now become `azuredevops.core` etc.
```

Comment:
Also, Python `service_endpoint` is now `serviceendpoint`.

Reply:
Ok, that specific one was an example but I will add a full list of them all to the changelog to be more explicit